src/components/AudioPlayer/AudioPlayer.stories.mdx

```diff
@@ -0,0 +1,189 @@
+import DSProvider from "../../theme/provider";
+
+<Meta
+  title={getCategory("AudioPlayer")}
+  component={AudioPlayer}
+  decorators={[withDesign]}
+  parameters={{
+    design: {
+      type: "figma",
+      url: "https://www.figma.com/file/qShodlfNCJHb8n03IFyApM/Main?node-id=36835%3A25338",
+    },
+    jest: ["AudioPlayer.test.tsx"],
+  }}
+  argTypes={{
+    audioType: {},
+    className: { control: false },
+    id: { control: false },
+  }}
+/>
+
+# Audio Player
+
+| Component Version | DS Version |
+| ----------------- | ---------- |
+| Added             | `1.1.0`    |
+| Latest            | `1.1.0`    |
+
+## Table of Contents
+
+- [Overview](#overview)
+- [Component Props](#component-props)
+- [Accessibility](#accessibility)
+- [Example Embed Code Snippets](#example-embed-code-snippets)
+- [Errored](#errored)
+
+## Overview
+
+<Description of={AudioPlayer} />
+
+The `AudioPlayer` component is used to embed a streaming audio player from 3rd party services such as `Libsyn`, `SoundCloud`, and `Spotify`
+on a page. The component requires the `embedCode` prop or `filePath` prop. The component will not function properly if you try to set
+both of them.
+
+**This version will not be used for the playback of locally hosted audio files**.
+
+## Component Props
+
+<Canvas withToolbar>
+  <Story
+    name="AudioPlayer"
+    args={{
+      className: undefined,
+      descriptionText:
+        "Audio description lorem ipsum dolor simet. Cum sociis natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Sed posuere consectetur est at lobortis.",
+      embedCode: `<iframe title="Libsyn Player" style="border: none" src="//html5-player.libsyn.com/embed/episode/id/18268511/height/90/theme/custom/thumbnail/yes/direction/backward/render-playlist/no/custom-color/87A93A/" height="90" width="100%" scrolling="no"  allowfullscreen webkitallowfullscreen mozallowfullscreen oallowfullscreen msallowfullscreen></iframe>`,
+      headingText: "Audio Title",
+      helperText: "Audio helper text lorem ipsum dolor simet.",
+      id: "audioplayer-id",
+      iframeTitle: "Libsyn Audio",
+      audioType: "libsyn",
+    }}
+  >
+    {(args) => (
+      <AudioPlayer
+        {...args}
+        embedCode={args.embedCode}
+        audioType={args.audioType && args.audioType}
+      />
+    )}
+  </Story>
+</Canvas>
+
+<ArgsTable story="AudioPlayer" />
+
+## Accessibility
+
+The `VideoPlayer` component renders an `iframe` element from `Libsyn`, `SoundCloud`, or `Spotify`.
+`iframe`s can be accessible if a `title` attribute is provided.
+The title must already be provided in the embed code value that is passed to the
+`embedCode` prop. A default one is provided but a custom one is recommended.
+
+To guarantee that the audio player will be accessible.
+Besides setting the `title` in the embedded `iframe`, there are _some_ ways we
+can reduce accessibility issues.
+
+- The player should not be set to auto-play when the page first loads.
```

Comment:
Remove the "autoplay" text in the embed code examples. This is great to have but the examples still have that code.